vega_sim/scenario/configurable_market/scenario.py

```diff
@@ -98,6 +101,7 @@ def configure_agents(
         tag: str,
         market_config: Optional[MarketConfig] = None,
```

Comment:
I think when I made this I should have made this a `kwarg` instead, might be good to change here?